src/api/repos/collaborators.rs

```diff
@@ -8,6 +8,8 @@ pub struct ListCollaboratorsBuilder<'octo, 'r> {
     per_page: Option<u8>,
     #[serde(skip_serializing_if = "Option::is_none")]
     page: Option<u32>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    permission: Option<String>,
```

Comment:
Unsure if we should use an enum for the possible values instead.

I'm not quite familiar with enum serialization.

Reply:
Is there a known limited set of values it should be? Then it should use an `enum`. You can copy one of the other enums in the crate as an example.

Reply:
Sounds reasonable. Let me update the code here in hours.

Reply:
Updated.